cmd/process-agent/main.go

```diff
@@ -24,8 +49,12 @@ func main() {
 	flag.StringVar(&opts.check, "check", "", "Run a specific check and print the results. Choose from: process, connections, realtime")
 	flag.Parse()
 
-	exit := make(chan struct{})
+	RootCmd.AddCommand(commands.Config(setupConfigClient))
```

Comment:
Mind sharing in the PR description what happens when you run with with `process-agent --help`?

Reply:
Added!

Reply:
Sure! Added